src/jit/lir.cpp

```diff
@@ -576,6 +576,7 @@ void LIR::Range::InsertBefore(GenTree* node, GenTree* insertionPoint)
         assert(LastNode() == nullptr);
 
         FirstNode() = node;
+        LastNode() = node;
```

Comment:
looks like the function comment needs to be updated to indicate that null insertionPoint is allowed, and what the semantics are.